.circleci/config.yml

```diff
@@ -288,7 +288,7 @@ jobs:
       - run:
           name: test op-e2e
           command: |
-            gotestsum --junitfile /test-results/op-e2e.xml -- -coverpkg=github.com/ethereum-optimism/optimism/... -coverprofile=coverage.out -covermode=atomic ./...
+            gotestsum --format standard-verbose --junitfile /test-results/op-e2e.xml -- -coverpkg=github.com/ethereum-optimism/optimism/... -coverprofile=coverage.out -covermode=atomic ./...
```

Comment:
Does this pass through flags to the go testing tool? In that case we might need to add `-parallel`

Reply:
It does pass through flags, but it's already running the tests in parallel. You can also see that parallel tests are being picked up in the CI job - the e2e tests are taking 30s rather than 3m.